source/deploy/desktop-app.rst

```diff
@@ -76,9 +76,9 @@ You can customize and distribute your own Mattermost Desktop application by conf
 Windows App: Pre-Configuration and Silent Deployment
 -----------------------------------------------------
 
-You can distribute the official Windows Desktop App silently to end users, pre-configured with the server URL. You can also set all the `app settings <https://docs.mattermost.com/help/apps/desktop-guide.html#app-options>`__ except for the **Start app on login** option.
+You can distribute the official Windows Desktop App silently to end users, pre-configured with the server URL. You can also set all the `app settings <https://docs.mattermost.com/messaging/managing-desktop-app-options.html>`__ except for the **Start app on login** option.
 
-1. Download the latest Windows installer from the `Mattermost download page <https://mattermost.com/download/#mattermostApps>`__.
+1. Download the latest Windows installer from the `Mattermost download page <https://mattermost.com/download/#>`__.
```

Comment:
Curious as to why the "mattermostApps" part was removed, as it appears to land in the correct section of the correct page.

Reply:
I would reply the same as above. Reverting this as well. Thanks.